dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -72,11 +73,17 @@
     @OrderBy("id")
     private SortedSet<Label> labels = new TreeSet<>();
 
-    @ManyToMany(fetch = FetchType.LAZY, cascade = { CascadeType.PERSIST, CascadeType.MERGE })
```

Comment:
If I recall correctly, it was deleted because it was causing an issue:  entry returns starredusers, those users return starred entries, those entries returning more users...etc.